src/Chapter2.hs

```diff
@@ -621,7 +629,9 @@ Write a function that takes elements of a list only in even positions.
 >>> takeEven [2, 1, 3, 5, 4]
 [2,3,4]
 -}
-takeEven = error "takeEven: Not implemented!"
+takeEven :: [Int] -> [Int]
+takeEven [] =[]
+takeEven(x:xs) = if mod x 2 == 0 then [x] ++ takeEven xs else [] ++ takeEven xs
```

Comment:
Here you also can avoid `++` operator by replacing it with `:` or no action at all!

```suggestion
takeEven(x:xs) = if mod x 2 == 0 then x : takeEven xs else takeEven xs
```